README.md

```diff
@@ -23,15 +23,17 @@ A single request and response pair is represented as shown in the below example:
       "user-agent": "Mozilla/5.0 (pc-x86_64-linux-gnu) Siege/3.0.8"
     },
     "pathname": "/user/repos",
-    "query": { "param": "value" }
+    "query": { "param": "value" },
+    "timestamp": "2018-11-13T20:20:39+01:00"
```

Comment:
Are milliseconds allowed by the `date-time` format?

Reply:
Yes - the ISO 8601 allows arbitrary precision, as in `2018-11-13T20:20:39+01:00.1234243234521345124`.

In practice implementations will round it, but I think microsecond should be safe across implementations.